src/content/ueber-uns/abteilungen.svx

```diff
@@ -1,57 +1,49 @@
 ---
 title: Abteilungen
-date: '2022-11-15'
+date: '2023-03-07'
 nav_index: 2
 ---
 
-## Allgemeines
+Die Task Force Hellcat besteht aus drei Abteilungen, die verschiedene Truppengattungen umfassen. Auf dieser Seite werden die Truppengattungen der Abteilungen näher erläutert.
 
-Die Task Force Hellcat ist in drei Abteilungen untergliedert, die jeweils durch einen Abteilungsleiter organisiert und verwaltet werden. Alle Abteilungen umfassen mehrere Truppengattungen, die auf dieser Seite erläutert werden.
+(TODO: Insert tfhc structure diagram here?)
```

Comment:
This line should be removed before merging. I think you could use a HTML comment if you want to keep that line in the file for longer,

```suggestion
<!-- (TODO: Insert tfhc structure diagram here?) -->
```

I believe that would not be rendered.

Reply:
I do not intend to keep that line for long. Since this PR will have to wait for merge until https://github.com/taskforcehellcat/wiki/pull/63 is merged, I'm hoping for the new structure diagram to be finished by then. If not, we can simply remove this line before merge and insert the diagram later on.